packages/core/src/input/wheel/WheelManager.ts

```diff
@@ -11,19 +11,28 @@ export class WheelManager implements IInput {
   _delta: Vector3 = new Vector3();
 
   private _nativeEvents: WheelEvent[] = [];
-  private _canvas: HTMLCanvasElement;
+  private _target: EventTarget;
   private _hadListener: boolean;
 
+  get target(): EventTarget {
+    return this._target;
+  }
+
+  set target(value: EventTarget) {
+    const preTarget = this._target;
+    if (preTarget !== value) {
+      preTarget && this._removeEventListener();
+      this._target = value;
+      value && this._addEventListener();
+    }
+  }
+
```

Comment:
Don't need open this， Open this need do some warp in cross platform engine
The historical baggage is large and there is no urgent need for the time being.

Reply:
Done.